lib/ddtrace/workers.rb

```diff
@@ -62,6 +63,19 @@ def callback_services
         end
       end
 
+      def callback_runtime_metrics
+        return true unless Datadog.runtime_metrics.send_stats?
+
+        begin
+          Datadog.runtime_metrics.flush
+        rescue StandardError => e
+          # ensures that the thread will not die because of an exception.
+          # TODO[manu]: findout the reason and reschedule the send if it's not
```

Comment:
what did manu do?! :)

assuming copy/pasta?

Reply:
Ah, Manu was writing Ruby again? Maybe I should do something about that... :roll_eyes: